### ISSUE TITLE
Transclude agenda buffer?

### ISSUE THREAD

**Issue by shlevy** [nobiot/org-transclusion#50]
I'm trying to create a "dashboard" org-roam note that includes things like my active projects, my daily agenda, etc. Would it be possible, whether within org-transclusion itself or reusing some of the core mechanisms, to include an org agenda buffer in the transcluded view?

**Comment by shlevy** [comment]
I haven't looked at the code, but I wonder if there's a set of primitives around transcluding specific *buffers* that can be exposed, and have the file transclusion be built on top of this...

**Comment by shlevy** [comment]
Would also be nice to be able to transclude things like "all backlinks to some specified org-roam note"

**Comment by nobiot** [comment]
I have a feeling that there is other packages that might be more suited for dashboards.
As I indicated in [README](https://github.com/nobiot/org-transclusion#intended-use), I have created Org-transclusion as a writing tool. It is fantastic that it can be used in other purposes I never dreamed of. I am not sure if the transclusion is a good tool for dashboard...

Is it not possible to use Org-agenda and/or [Org-super-agenda](https://github.com/alphapapa/org-super-agenda) as the better foundation for your dashboard, perhaps? Alternatively, this is not a dashboard, but might get you started in the right path ([Delve](https://github.com/publicimageltd/delve))
